apps/stellar-contracts/tests/cli/deploy.sh

```
@@ -0,0 +1,211 @@
+# Create deployment report
+log_info "Creating deployment report..."
+cat > "$TESTS_DIR/logs/deployment-report-$(date +%Y%m%d-%H%M%S).json" << EOF
+{
+  "deployment_timestamp": "$(date -u +"%Y-%m-%dT%H:%M:%SZ")",
+  "network": "$STELLAR_NETWORK",
+  "account_id": "$ACCOUNT_ID",
+  "account_balance": "$BALANCE",
+  "contracts": {
+EOF
+
+for i in "${!CONTRACTS[@]}"; do
```

Shellcheck (0.11.0)

[error] Couldn't parse this for loop. Fix to allow more checks.

(SC1073)